<launch>
  <arg name="cam_width" default="1280" />
  <arg name="cam_height" default="960" />
  <arg name="cam_model" default="daA1280-54uc" />
  <arg name="camera_frame" default="camera" />

  <!-- Choose the calibration file that corresponds to the resolution being used. -->
  <arg name="cam_right_calibration_file" default="file://$(find basler_camera)/config/right_unrectified.yaml"/>
  <arg name="cam_left_calibration_file" default="file://$(find basler_camera)/config/left_unrectified.yaml"/>

  <node ns="stereo" name="right" pkg="basler_camera" type="basler_camera_node" output="screen">
    <param name="frame_rate" type="int" value="10" />
    <param name="frame_id" type="string" value="right_camera" />
    <param name="camera_info_url" type="string" value="$(arg cam_right_calibration_file)" />
    <param name="serial_number" type="string" value="21741659"/>
    <rosparam command="load" file="$(find basler_camera)/config/dart_sync_slave.yaml"/>
  </node>

  <node ns="stereo" name="left" pkg="basler_camera" type="basler_camera_node" output="screen">
    <param name="frame_rate" type="int" value="10" />
    <param name="frame_id" type="string" value="left_camera" />
    <param name="camera_info_url" type="string" value="$(arg cam_left_calibration_file)" />
    <param name="serial_number" type="string" value="21741660"/>
    <rosparam command="load" file="$(find basler_camera)/config/dart_sync_master.yaml"/>
  </node>

</launch>
